BMSUtils.cpp

```diff
@@ -0,0 +1,108 @@
+#include <iostream>
+#include <assert.h>
+#include "BMSUtils.h"
+
+
+Parameter::Parameter(Language Lang, float min_value, float max_value, string name)
+{
+    min_threshold = min_value;
+    max_threshold = max_value;
+    parameter_name = name;
+    tolerance = max_threshold * 0.05 ;
+    if(Lang == ENGLISH){
```

Comment:
The complexity / length of this function increases as languages are increased. Try separating the data into an array instead of keeping it in-line within the code.